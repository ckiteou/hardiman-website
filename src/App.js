import React from 'react'
import Navbar from './components/home/Navbar'
import './styles/hardiman.scss'


const App = ({children}) => (
  <div className="hardiman_website">
    <header className="header">
      <Navbar/>
    </header>
    <div className="main">
      {children}
    </div>
  </div>
)

export default App;
