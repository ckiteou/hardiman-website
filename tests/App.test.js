import React from 'react';
import renderer from 'react-test-renderer'

import App from '../src/App';

describe('App', () => {
  it('renders', () => {
    const tree = renderer.create(
      <App />
    ).toJSON();
    expect(tree).toMatchSnapshot()
  });
});
