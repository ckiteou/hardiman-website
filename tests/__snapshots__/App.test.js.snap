// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
<div
  className="hardiman_website"
>
  <header
    className="header"
  >
    <nav
      className="nav_bar"
    >
      <a
        className="nav_link"
        onClick={[Function]}
        style={Object {}}
      >
        Home
      </a>
      <a
        className="nav_link"
        onClick={[Function]}
        style={Object {}}
      >
        About
      </a>
      <a
        className="nav_link"
        onClick={[Function]}
        style={Object {}}
      >
        Services
      </a>
      <a
        className="nav_link"
        onClick={[Function]}
        style={Object {}}
      >
        Portfolio
      </a>
      <a
        className="nav_link"
        onClick={[Function]}
        style={Object {}}
      >
        Contact
      </a>
    </nav>
  </header>
</div>
`;
